scripts/api/data/metadatablocks/codemeta.tsv

```diff
@@ -12,20 +12,27 @@
 	targetProduct	Target Product	Target Operating System / Product to which the code applies. If applies to several versions, just the product name can be used.		text	8	#VALUE	TRUE	FALSE	TRUE	TRUE	FALSE	FALSE		codeMeta20	https://schema.org/targetProduct
 	buildInstructions	Build Instructions	Link to installation instructions/documentation	e.g. https://github.com/user/project/blob/main/BUILD.md	url	9	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	TRUE	FALSE	FALSE	FALSE		codeMeta20	https://codemeta.github.io/terms/buildInstructions
 	softwareRequirementsItem	Software Requirements	Required software dependencies		none	10		FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		codeMeta20
-	softwareRequirements	Name & Version	Name and version of the required software/library dependency	e.g. Pandas 1.4.3	text	0	#VALUE	TRUE	FALSE	FALSE	FALSE	TRUE	FALSE	softwareRequirementsItem	codeMeta20	https://schema.org/softwareRequirements
+	softwareRequirements	Name & Version	Name and version of the required software/library dependency	e.g. Pandas 1.4.3	text	0	#VALUE	TRUE	FALSE	FALSE	FALSE	TRUE	TRUE	softwareRequirementsItem	codeMeta20	https://schema.org/softwareRequirements
 	softwareRequirementsInfoUrl	Info URL	Link to required software/library homepage or documentation (ideally also versioned)	e.g. https://pandas.pydata.org/pandas-docs/version/1.4.3	url	1	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	TRUE	FALSE	softwareRequirementsItem	codeMeta20	https://dataverse.org/schema/codeMeta20/softwareRequirementsInfoUrl
+	softwareRequirementsUrl	Download URL	Link to required software/library	https://...	url	2	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	softwareRequirementsItem	codeMeta20	
```

Comment:
This is a new field, right? Can you please add it to src/main/java/propertyFiles/codeMeta20.properties? And please make any tweaks to that file to keep it the same as the tsv file.

This is probably why the "Properties Check / Metadata Blocks Properties (pull_request)" check is failing by the way. Here's a screenshot from the [latest run](https://github.com/IQSS/dataverse/actions/runs/12282558382/job/34274040705?pr=11087):

![Screenshot 2025-01-30 at 1 28 41 PM](https://github.com/user-attachments/assets/e7225438-5b92-46b2-aa9b-f4b67ed03b42)

Reply:
@pdurbin done, properties file is actualised, checks run fine